src/config/prismLight.js

```diff
@@ -86,10 +90,16 @@ module.exports = {
         color: '#6F42C1',
       },
     },
+    {
```

Comment:
taken from here: https://github.com/FormidableLabs/prism-react-renderer/blob/9c041dfd8580b8b71f55c2a57e61f4c63356d4f4/packages/prism-react-renderer/src/themes/github.ts#L34-L50

and modified the color.